src/component/mxgraph/BpmnCellRenderer.ts

```diff
@@ -30,7 +30,7 @@ export class BpmnCellRenderer extends mxgraph.mxCellRenderer {
       dict = new mxgraph.mxDictionary<mxShape>();
 
       for (const currentOverlay of overlays) {
-        const shape = state.overlays != null ? state.overlays.remove(currentOverlay) : null;
+        const shape = state.overlays == null ? null : state.overlays.remove(currentOverlay);
```

Comment:
**question:** this is mainly a warning for the next estlint rules we will enabled. The question is: "@csouchet do do you agree with the following statement?" 

Here, we modify some original mxGraph code. Up to now, we've tried to modify the code only when necessary. This was initially done to manage mxGraph version changes (know what we've changed, so if the original mxGraph code changes, we can more easily backport our changes to the new implementation).
With mxGraph at the end of its life, this is probably no longer a strong requirement. However, if we decide to switch to maxGraph, it may be useful for the initial migration. The ability to modify maxGraph will be resurrected.

Here, we're starting to change something that isn't necessary, except to match our linting rules (we've done it before, but that was for indentation, or switching to TypeScript, but only very slight things).
The change here doesn't alter the effect of the implementation: it just reverses the condition. So, in this case, I think it's safe to apply the change.
This may not be the case for other types of changes due to the next unicorn rules we may activate.

Reply:
@tbouffard  hadn't paid attention to the changes here, but indeed, for other rules, I disabled them on the `mxgraph` code, because I didn't know the impact, for example replacing `null` with `undefined`.